squidb/src/com/yahoo/squidb/sql/Table.java

```diff
@@ -130,6 +130,18 @@ public void appendCreateTableSql(@Nonnull CompileContext compileContext, @Nonnul
     }
 
     /**
+     * @return the <code>CREATE TABLE</code> statement for creating this table. Users should generally not need to call
+     * this method directly unless they are not working with a SquidDatabase instance and wish to create tables
+     * manually.
+     */
+    @Nonnull
+    public String getCreateTableSql() {
```

Comment:
I wonder if these should take a `CompileContext` argument instead of using a fixed SQLite version code.